api/db-cdr/generate-cdr/create-tables.sh

```diff
@@ -37,6 +37,46 @@ do
+    elif [[ "$table_name" == 'ds_condition_occurrence' && "$TABLE_LIST" == *'visit_detail'* ]]
+    then
+      echo "Creating $table_name"
+      bq --quiet --project_id="$BQ_PROJECT" mk --schema="$schema_path/$json_name" --time_partitioning_type=DAY --clustering_fields person_id,domain "$BQ_DATASET.$table_name"
+    elif [[ "$table_name" == 'ds_condition_occurrence_52' && "$TABLE_LIST" =~ *'visit_detail'* ]]
+    then
+      echo "Creating $table_name"
+      bq --quiet --project_id="$BQ_PROJECT" mk --schema="$schema_path/$json_name" --time_partitioning_type=DAY --clustering_fields person_id,domain "$BQ_DATASET.$table_name"
+    elif [[ "$table_name" == 'ds_drug_exposure' && "$TABLE_LIST" == *'visit_detail'* ]]
+    then
+      echo "Creating $table_name"
+      bq --quiet --project_id="$BQ_PROJECT" mk --schema="$schema_path/$json_name" --time_partitioning_type=DAY --clustering_fields person_id,domain "$BQ_DATASET.$table_name"
+    elif [[ "$table_name" == 'ds_drug_exposure_52' && "$TABLE_LIST" =~ *'visit_detail'* ]]
+    then
+      echo "Creating $table_name"
+      bq --quiet --project_id="$BQ_PROJECT" mk --schema="$schema_path/$json_name" --time_partitioning_type=DAY --clustering_fields person_id,domain "$BQ_DATASET.$table_name"
+    elif [[ "$table_name" == 'ds_measurement' && "$TABLE_LIST" == *'visit_detail'* ]]
+    then
+      echo "Creating $table_name"
+      bq --quiet --project_id="$BQ_PROJECT" mk --schema="$schema_path/$json_name" --time_partitioning_type=DAY --clustering_fields person_id,domain "$BQ_DATASET.$table_name"
+    elif [[ "$table_name" == 'ds_measurement_52' && "$TABLE_LIST" =~ *'visit_detail'* ]]
+    then
+      echo "Creating $table_name"
+      bq --quiet --project_id="$BQ_PROJECT" mk --schema="$schema_path/$json_name" --time_partitioning_type=DAY --clustering_fields person_id,domain "$BQ_DATASET.$table_name"
+    elif [[ "$table_name" == 'ds_observation' && "$TABLE_LIST" == *'visit_detail'* ]]
+    then
+      echo "Creating $table_name"
+      bq --quiet --project_id="$BQ_PROJECT" mk --schema="$schema_path/$json_name" --time_partitioning_type=DAY --clustering_fields person_id,domain "$BQ_DATASET.$table_name"
+    elif [[ "$table_name" == 'ds_observation_52' && "$TABLE_LIST" =~ *'visit_detail'* ]]
+    then
+      echo "Creating $table_name"
+      bq --quiet --project_id="$BQ_PROJECT" mk --schema="$schema_path/$json_name" --time_partitioning_type=DAY --clustering_fields person_id,domain "$BQ_DATASET.$table_name"
+    elif [[ "$table_name" == 'ds_procedure_occurrence' && "$TABLE_LIST" == *'visit_detail'* ]]
+    then
+      echo "Creating $table_name"
+      bq --quiet --project_id="$BQ_PROJECT" mk --schema="$schema_path/$json_name" --time_partitioning_type=DAY --clustering_fields person_id,domain "$BQ_DATASET.$table_name"
+    elif [[ "$table_name" == 'ds_procedure_occurrence_52' && "$TABLE_LIST" =~ *'visit_detail'* ]]
```

Comment:
Please remove these conditions. We only need to add a condition for ds_procedure_occurrence and it should only be created in the default else -> must be unclustered.

Reply:
I did not follow what this means, but ok. removed all the condition-lines above